src/devnet/integrated-devnet-logger.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from "path";
+import * as fs from "fs";
+
+export abstract class IntegratedDevnetLogger {
+
+    // Promisify fs
+    private fsPromises = fs.promises;
```

Comment:
It was only because that `fsPromises` was accessed in the class. https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/179/commits/cfd12c44d12d31196d3a51293bdf70b855faa9fe